app/src/main/res/values/strings.xml

```diff
@@ -276,71 +277,71 @@
     <string name="settings_general_users_key">ManageUsers</string>
     <string name="settings_general_users_title">Manage users</string>
     <string name="settings_general_users_summary">Add/remove users for protected topics</string>
     <string name="settings_general_users_prefs_title">Users</string>
     <string name="settings_general_users_prefs_user_not_used">Not used by any topics</string>
-    <string name="settings_general_users_prefs_user_used_by_one">Used by topic %1$s</string>
-    <string name="settings_general_users_prefs_user_used_by_many">Used by topics %1$s</string>
+    <string name="settings_general_users_prefs_user_used_by_one">Used by \"%1$s\" topic</string>
+    <string name="settings_general_users_prefs_user_used_by_many">Used by \"%1$s\" topics</string>
     <string name="settings_general_users_prefs_user_add">Add users</string>
     <string name="settings_general_users_prefs_user_add_title">Add new user</string>
     <string name="settings_general_users_prefs_user_add_summary">Create a new user for a new server</string>
     <string name="settings_general_dark_mode_key">DarkMode</string>
     <string name="settings_general_dark_mode_title">Dark mode</string>
-    <string name="settings_general_dark_mode_summary_system">Use the system default</string>
-    <string name="settings_general_dark_mode_summary_light">Light mode is enabled</string>
-    <string name="settings_general_dark_mode_summary_dark">Dark mode is enabled. Are you a vampire?</string>
+    <string name="settings_general_dark_mode_summary_system">Using the system default</string>
+    <string name="settings_general_dark_mode_summary_light">Light mode on</string>
+    <string name="settings_general_dark_mode_summary_dark">Dark mode on. Are you a vampire?</string>
     <string name="settings_general_dark_mode_entry_system">Use system default</string>
     <string name="settings_general_dark_mode_entry_light">Light mode</string>
     <string name="settings_general_dark_mode_entry_dark">Dark mode</string>
-    <string name="settings_backup_restore_header">Backup &amp; Restore</string>
+    <string name="settings_backup_restore_header">Backups and restoration</string>
     <string name="settings_backup_restore_backup_key">Backup</string>
     <string name="settings_backup_restore_backup_title">Backup to file</string>
     <string name="settings_backup_restore_backup_summary">Export config, notifications and users</string>
     <string name="settings_backup_restore_backup_entry_everything">Everything</string>
     <string name="settings_backup_restore_backup_entry_everything_no_users">Everything, except users</string>
     <string name="settings_backup_restore_backup_entry_settings_only">Settings only</string>
-    <string name="settings_backup_restore_backup_successful">Backup successful</string>
+    <string name="settings_backup_restore_backup_successful">Backup made</string>
```

Comment:
No :-(

Reply:
With "successful", the idea that a backup could be made in an unsuccessful manner is possible.
It just irks me.